boopmain.ino

```diff
@@ -128,10 +142,20 @@ void loop()
     draw();
     vibe_flash();
     last_draw = millis();
-        
-    if(color_select > 6) {
+  }
+
+//Color change if button is released
+  if(is_button_pressed == false & last_buttonpress == true) {
+    color_select++;
+  }
+  //Update last button press
+  if(is_button_pressed){
+    last_buttonpress = true;
+} else {
+    last_buttonpress = false;
+}
```

Comment:
```suggestion
last_buttonpress = is_button_pressed;
```

don't need the if here. you want the value of last_buttonpress to be is_button_pressed